compair/static/modules/answer/answer-form-partial.html

```diff
@@ -41,14 +41,17 @@ <h2><i class="fa fa-comments"></i> {{assignment.name}}</h2>
                 <p>Allowed file types for attachments include {{ UploadValidator.getAttachmentExtensionsForDisplay() }}, with a maximum file size of <strong>{{UploadValidator.getAttachmentUploadLimit() | filesize }}</strong>. Any files you upload will be attached when you <strong>save or submit this answer</strong> but not before.</p>
         </div>
 
-        <div class="form-group" ng-if="canManageAssignment && answer.draft && comparison_example !== true">
+        <div class="form-group" ng-if="canManageAssignment && answer.draft && comparison_example !== true" ng-class="{'has-error': answerForm.user_id.$invalid}">
             <label class="required-star" for="user_id">Submit As</label>
-            <select id="user_id" name="user_id" required
+            <select id="user_id" name="user_id" ng-required="validated"
```

Comment:
instead of using a `validated`variable flag that you need to track and manage you can use `answerForm.$submitted` which will be true after a submit button is pressed for the form